src/appsco-app.html

```diff
@@ -1423,6 +1433,14 @@
                 api-errors="[[ _apiErrors ]]"
                 on-customer-added="_onCustomerAdded"></appsco-add-customer>
 
+        <appsco-add-partner-admin
```

Comment:
@brankastanic already implemented the sam action for list of customers :). Maybe we should have one component only.